README.md

```diff
@@ -2,7 +2,6 @@
 
 # MATH-MAGICIANS
 
```

Comment:
-  Kindly add a description for your project.

Reply:
Thank you, I did that.